lighthouse-core/driver/index.js

```diff
@@ -100,15 +100,19 @@ class Driver {
     const loadData = {};
     let pass = Promise.resolve();
 
+    if (config.traceName) {
+      loadData[config.traceName] = {};
```

Comment:
I also wonder if trace data should live at the top level of artifacts. Easy to stomp on artifacts without realizing it. Nested like `artifacts.tracingData[traceName]`, maybe?